src/components/overview/chart/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { colors } from '../../../theme';
+import { data } from './helper';
+import {
+  Area,
+  AreaChart,
+  CartesianGrid,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import { FunctionComponent, useState } from 'react';
+const Chart = () => {
+  const currentDate = new Date();
+  const currentHour = currentDate.getHours();
+  const [isTodayHovered, setIsTodayHovered] = useState(false);
+  const [isYesterdayHovered, setIsYesterdayHovered] = useState(false);
+  interface IProps {
+    cx: number;
+    cy: number;
+    gray?: boolean;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const CustomizedDot: FunctionComponent<IProps & any> = ({ cx, cy, gray }) => {
+    return (
+      <svg x={cx - 13} y={cy - 13} height='100' width='100'>
+        <circle
+          cx='13'
+          cy='13'
+          r='5'
+          stroke={gray ? `${colors.grayDark}` : `${colors.primaryBlue}`}
+          strokeWidth='4'
+          fill='white'
+        />
+        <circle
+          cx='13'
+          cy='13'
+          r='12'
+          stroke={gray ? `${colors.grayDark}` : `${colors.primaryBlue}`}
+          opacity='0.16'
+          strokeWidth='2'
+          fill='transparent'
+        />
+      </svg>
+    );
+  };
+  const filteredData = data.map((entry) => ({
+    ...entry,
+    Today: entry.Today && parseInt(entry.hours) < currentHour ? entry.Today : null,
+    Yesterday: entry.Yesterday && parseInt(entry.hours) < currentHour ? entry.Yesterday : null,
+  }));
+
+  return (
+    <ResponsiveContainer width='100%' height={380}>
+      <AreaChart data={filteredData} margin={{ top: 20, right: 30, left: 5 }}>
+        <defs>
+          <linearGradient id='colorToday' x1='0' y1='0' x2='1' y2='0'>
+            <stop offset='5%' stopColor={colors.primaryBlue} stopOpacity={0.2} />
+            <stop offset='95%' stopColor={colors.primaryBlue} stopOpacity={0} />
+          </linearGradient>
+
+          <linearGradient id='colorYesterday' x1='0' y1='0' x2='1' y2='0'>
+            <stop offset='5%' stopColor={colors.grayLight} stopOpacity={0.5} />
+            <stop offset='95%' stopColor={colors.grayLight} stopOpacity={0} />
+          </linearGradient>
+        </defs>
+        <CartesianGrid vertical={false} stroke={colors.graphDivider} />
+        <XAxis
+          dataKey='hours'
+          tickLine={false}
+          axisLine={false}
+          tickMargin={12}
+          tickSize={6}
+          tick={{ fontSize: 10, fill: colors.primaryGray }}
+          padding={{ right: 15 }}
+        />
+        <YAxis
+          orientation='right'
+          type='number'
+          tickLine={false}
+          tick={{ fontSize: 10, fill: colors.primaryGray }}
+          ticks={[0, 10, 20, 30, 40, 50, 60]}
```

Comment:
Let's move all axes values and styles objects to the helper.ts and styled.ts

Reply:
fix: values objects -> helper.ts, style objects -> styled.ts